### PR TITLE
More columns&filters in supplier invoices

### PR DESCRIPTION
**Pull request by a-schild**

### CODE REVIEW

File: htdocs/fourn/facture/paiement.php [hunk 1]
```diff
@@ -424,8 +435,8 @@
 	        // Bouton Enregistrer
 	        if ($action != 'add_paiement')
 	        {
-				print '<br><div class="center"><input type="checkbox" checked name="closepaidinvoices"> '.$langs->trans("ClosePaidInvoicesAutomatically");
-				print '<br><input type="submit" class="button" value="'.$langs->trans('Save').'"></div>';
+				print '<center><br><input type="checkbox" checked="checked" name="closepaidinvoices"> '.$langs->trans("ClosePaidInvoicesAutomatically");
```
Comment (aspangaro) [inline htdocs/fourn/facture/paiement.php]: Balise "center" are deprecated in html5. Please leave the div balise with the class "center".

**Comment by a-schild** [comment]
Sorry, mixed up the  3.7 version of the file. Is now corrected

**Comment by eldy** [comment]
Using sql datediff and now is forbidden on sql side. You must convert code to do same but with php.
"date < '".$db->idate(dol_now + offset)."'"

**Comment by a-schild** [comment]
@eldy 
Hmmm..
where is this documented, so I may find other forbidden things?
Currently I wish the DB to calculate the difference between two dates in days, so we see how much the payment is overdue.
The "date < '".$db->idate(dol_now + offset)."'" won't help me, unless I then calculate/format it when displaying the result in the html table.

What's the correct way to do this?

**Comment by rdoursenaud** [comment]
@a-schild Good point. Documentation needed!

**Comment by eldy** [comment]
Among the 2 dates,there is the NOW sql function.
So the correct way is the one i suggested in my first post. 
The sql function NOW must be replaced with the php functuon dol_now and the datediff must be replaced by a php addition.
Rules are here: http://wiki.dolibarr.org/index.php/Language_and_development_rules#SQL_Coding_rules

I added examples on wiki page.

**Pull request closed without merging**